specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2023-01-01-preview/hybridCloud.json

```diff
@@ -0,0 +1,1002 @@
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "virtualHub": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The virtualHub to which the cloud connection belongs.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "sharedKey": {
+          "type": "string",
+          "description": "Shared key of the cloud connection.",
+          "x-ms-secret": true
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the cloud collection resource.",
+          "readOnly": true
+        }
+      },
+      "description": "Cloud connection resource properties."
+    },
+    "CloudConnection": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "description": "Resource which represents the managed network connection between Azure Gateways and remote cloud gateways.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudConnectionPropertiesFormat",
+          "description": "Properties of the cloud connection."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ResourceFilter": {
+      "type": "object",
+      "description": "Remote Cloud Resource filter.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Resource name to filter."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type to filter."
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location to filter."
+        },
+        "tag": {
+          "type": "string",
+          "description": "Resource tags to filter in key:value format."
+        }
+      }
+    },
+    "DiscoveredResources": {
+      "type": "object",
+      "description": "API to discover resources in remote cloud referred by the connector.",
+      "properties": {
+        "resourceFilter": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ResourceFilter",
+          "description": "Remote cloud resource filter."
+        },
+        "discoveredResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiscoveredResource"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "List of discovered remote cloud resources."
+        }
+      }
+    },
+    "DiscoveredResource": {
```

Comment:
>DiscoveredResource [](http://example.com/codeflow?start=5&length=18)

Add Description property for DiscoveredResource definition.

Reply:
Fixed